_pages/en/web-services-access.md

```diff
@@ -21,57 +21,99 @@ This Policy applies only to unclassified assets (data, information, code, ..).
+ESDC is currently blocking access to specific Web 2.0 collaboration tools like Google Docs, Google Hangouts and OneDrive.
+Even though TBS policy generally recommends open access to these tools, ESDC's rationale for blocking them has been primarily to prevent accidental leak of protected information from employees.
+This is a valid concern that must be mitigated with a balanced approach.
 
-Technical Architecture has started to include some SaaS in the ESDC Technology Bricks (GitHub, Trello and Slack), but these don't reflect what can be accessed on the ESDC network or what is currently being used by employees (See [Appendix B](#appendix-b---saas-already-unblocked)).
+The proposed risk mitigation would be to augment the current Data Loss Prevention (DLP) by adding a tool that monitors and enforces classification of files and emails before they can be transmitted on the Internet.
+However, as the evaluated options so far can be costly, implementation is not being considered for the short term.
 
-The value add of technology bricks and technology standards regarding SaaS needs to be re-evaluated by considering the shared responsibility model of SaaS.
-More specifically that the maintenance and upkeep efforts of SaaS does not impact ESDC.
+Following current procedure, the additional SaaS tools would need to be added to ESDC Technology Bricks (alongside other approved SaaS like GitHub, Trello and Slack).
+However, the value add of adding SaaS to Technology Bricks and Standards should be re-evaluated by considering the shared responsibility model of SaaS.
+For instance, the maintenance and upkeep efforts of SaaS does not impact ESDC.
+
+Finally, there is a discrepancy between the approved SaaS and what can actually be accessed on the network by employees (See [Appendix B](#appendix-b---saas-already-unblocked)).
 
 ### Considerations
 
 More departments are using SaaS to collaborate and engage with citizens and other public administrations in Canada and the world.
-For example, the latest Open Government Plan and the Service and Digital Target Architecture from TBS were shared as drafts for consultation using Google Docs.
+For example, the latest Open Government Plan and the Service and Digital Target Architecture from TBS were shared as drafts for consultation and comments on Google Docs.
 The GC Digital Academy and Free Agents, as well as other groups, frequently use Google Forms for registration.
 
+In order to move forward, we propose a balanced approach in opening access to additional SaaS tools (See [Appendix A](#appendix-a---saas-to-enable)) to IITB employees in the short term, as a step toward more broader access in ESDC.
+As IITB employees generally don't deal with protected information and are well informed and trained on the management of information, we anticipate the risk to be minimal.
```

Comment:
I think that this is an assumption: DBAs most likely would have access to the data directly. Developers may be able to query DBs with sensitive information/data.

Reply:
Indeed, thus why I included the "well informed and trained" part ;)
I assume that DBAs are well trained to deal with protected information and unlikely to just copy/paste them into a random SaaS.

Reply:
OK, I see what you mean. My point was around trying to frame the "consideration" of access controls. DLP for email is a bit narrow in scope. Technically, there probably should be access audits and automated monitoring as well as a proposed way of mitigating the risks? Say for example, Database Access Monitoring, Application Activity Monitoring, etc. so that we increase our security and confidence in data loss prevention at the source, not at the perimeter level.
Quick read: https://opensource.com/article/19/4/log-analysis-tools